skactiveml/pool/tests/test_cal.py

```diff
@@ -0,0 +1,166 @@
+import unittest
+
+import numpy as np
+from sklearn.linear_model import LogisticRegression
+from sklearn.svm import SVC
+from sklearn.datasets import make_blobs
+from skactiveml.pool import Cal
+from skactiveml.classifier import ParzenWindowClassifier, SklearnClassifier
+from skactiveml.utils import MISSING_LABEL
+from skactiveml.tests.template_query_strategy import (
+    TemplateSingleAnnotatorPoolQueryStrategy,
+)
+from skactiveml.pool.tests.test_badge import (
+    ParzenWindowClassifierEmbedding,
+    ParzenWindowClassifierTuple,
+)
```

Comment:
If we use it here as well, we might want to move it into the test.utils subpackage